openfl/utilities/mocks.py

```diff
@@ -13,3 +13,9 @@ def __init__(self, feature_shape):
 
     def get_feature_shape(self):
         return self.feature_shape
+
```

Comment:
not related to this PR, but will it be cleaner to make 
MockDataLoader(DataLoader) and default other overrides for typing it better?

Reply:
We probably need more discussion how to move forward with `data_loader` in context of util operations without data present in the workspace.